### ISSUE TITLE
How can I move objects in the arcore-android-sdk example

### ISSUE THREAD

**Issue by arifnadeem7** [google-ar/arcore-android-sdk#63]
I can place Objects on tap, how can I move those Objects around? Can this be demonstrated in the example?

**Comment by Ketsuko** [comment]
From what I know, touching object with OpenGL itself is very tricky. That's why I've been trying to use ARCore with other OpenGL based 3D engines like Rajawali.

**Comment by arifnadeem7** [comment]
I was not specific with my question.

I wanted to know that since the **.obj** files are loaded with help of **ObjectRenderer** and since they are not **View's** how should we go about touching and interacting with them, I understand that you want to keep these examples to a basic level, but basics should also include interaction with the said objects. Right now the sample doesn't give a complete picture of how to go about developing an ARCore app other than loading a .obj model and placing it on a plane.

On the other hand ArKit has a comprehensive guide on how to go about interacting with objects [ArKit](https://developer.apple.com/documentation/arkit/handling_3d_interaction_and_ui_controls_in_augmented_reality), I wanted to know if perhaps we could have something like this in the future?

**Comment by stetro** [comment]
Hi, I think any kind of "touch" functionality is not the responsibility of ARCore since it depends on the fact what graphics framework you're using. For example in case of Rajawali you could use the Raypicker functionality. If you're on plain OpenGL like in the example here, you need to implement the frustum raycast on your own.

**Comment by arifnadeem7** [comment]
Thanks for the reply, I have never worked with OpenGL and the concepts that I came across seemed overwhelming and are very complicated.

I'll start looking into the frustum raycast approach but I hope that in future ARCore like ARKit will include utility methods for basic user interaction.

**Comment by robtakefman** [comment]
Agreed with anad7, Opengl shouldn't be a roadblock, if we're trying to roll this out to Android developers as a whole
